app/views/handlers/feedback.py

```diff
@@ -58,7 +63,27 @@ def get_page_title(self):
         return self.PAGE_TITLE
 
     def handle_post(self):
+        session_data = self._session_store.session_data
         self._session_store.session_data.feedback_count += 1
+
+        data = {
+            "feedback_text": self.form.data["feedback-type"],
+            "feedback_topic": self.form.data["feedback-text"],
+        }
+        metadata = {
+            "feedback_count": session_data.feedback_count,
+            "form_type": self._schema.form_type,
+            "language_code": session_data.language_code,
+            "object_key": str(uuid4()),
+            "region_code": self._schema.region_code,
+            "tx_id": session_data.tx_id,
+        }
+
+        feedback_upload = current_app.eq["feedback"].upload(data, metadata)
```

Comment:
`data` should be a superset of `metadata`, so like:
```py
        metadata = {
            "feedback_count": session_data.feedback_count,
            "form_type": self._schema.form_type,
            "language_code": session_data.language_code,
            "object_key": str(uuid4()),
            "region_code": self._schema.region_code,
            "tx_id": session_data.tx_id,
        }
        data = {
            "feedback_text": self.form.data["feedback-type"],
            "feedback_topic": self.form.data["feedback-text"],
            **metadata
        }
```

Reply:
Not sure why we would do this

Reply:
Because that is one of our requirements mentioned on the card.
- `Add all non form data attributes as metadata and payload. Form data to be payload only. `

Reply:
It's not how I read this, however I will change it for a easy life